src/components/calendar/DateCalendar.tsx

```diff
@@ -61,13 +63,15 @@ const DateCalendar = () => {
   } else if (dDayRaw < 0) {
     dDayMessage = (
       <>
-        결혼식이 <span css={dDayStyle}>{dDay}</span>일 지났어요
+        결혼식이<span css={dDayStyle}>{dDay}</span>일 지났어요
```

Comment:
```suggestion
        {basicInfo.maleName}
        <ImHeart color="#E0E0E0" css={heartIconStyle} size={11} />
        {basicInfo.femaleName} 행복한 결혼식을 마쳤습니다
```
변경 부탁드립니다!


```
// before
dDayMessage = '🎉 오늘은 결혼식 날이에요!';
```

```
// after
dDayMessage = (
  <>
    {basicInfo.maleName}
    <ImHeart color="#87BBBA" css={heartIconStyle} size={11} />
    {basicInfo.femaleName}의 행복한 결혼식 날이에요
  </>
);
```
->  이것도 변경 부탁드립니다!